sdk/servicebus/azure-messaging-servicebus/src/test/java/com/azure/messaging/servicebus/administration/ServiceBusAdministrationAsyncClientTest.java

```diff
@@ -90,10 +93,18 @@
     private Response<Object> objectResponse;
     @Mock
     private Response<Object> secondObjectResponse;
+    @Mock
+    private TokenCredential credential;
+    private AutoCloseable mockClosable;
 
     private final String queueName = "some-queue";
     private final String responseString = "some-xml-response-string";
     private final String secondResponseString = "second-xml-response";
+    private final String validToken = "some-valid-token";
+    private final String dummyEndpoint = "endpoint.servicebus.foo";
+    private final String forwardToEntity = "forward-to-entity";
+    private final String serviceBusSupplementaryAuthorizationHeaderName = "ServiceBusSupplementaryAuthorization";
```

Comment:
Reuse constant names? maybe if they are in the ServiceBusConstants class.

Reply:
Good idea, moved them to serviceBusConstants